VAMobile/src/constants/analytics.ts

```diff
@@ -635,25 +635,33 @@ export const Events = {
       name: 'vama_gender_id_success',
     }
   },
-  // vama_feedback_page_entered: (): Event => {
-  //   return {
-  //     name: 'vama_feedback_page_entered',
-  //   }
-  // },
-  // vama_feedback_page_closed: (): Event => {
-  //   return {
-  //     name: 'vama_feedback_page_closed',
-  //   }
-  // },
-  // vama_feedback_submitted: (taskCompleted: string, satisfaction: string): Event => {
-  //   return {
-  //     name: 'vama_feedback_submitted',
-  //     params: {
-  //       taskCompleted,
-  //       satisfaction,
-  //     },
-  //   }
-  // },
+  vama_feedback_ask: (screen: string, response: boolean): Event => {
+    return {
+      name: 'vama_feedback_ask',
+      params: {
+        screen,
+        response,
+      },
+    }
+  },
+  vama_feedback_closed: (screen: string): Event => {
+    return {
+      name: 'vama_feedback_page_exit',
```

Comment:
Any reason the event name differs from the function name? Usually they'd be the same

Reply:
Because it was originally exit and was too long and i forgot to change the name lol editing now.